app/src/main/java/com/sun/noteapp/data/source/local/NoteDatabase.kt

```diff
@@ -0,0 +1,85 @@
+package com.sun.noteapp.data.source.local
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import com.sun.noteapp.data.model.Note
+
+class NoteDatabase(private val context: Context)
+    :SQLiteOpenHelper(context, DATABASE_NAME, null, DATABASE_VERSION){
+
+    override fun onCreate(p0: SQLiteDatabase?) {
+        p0?.execSQL(SQL_CREATE_ENTRIES)
+    }
+
+    override fun onUpgrade(p0: SQLiteDatabase?, p1: Int, p2: Int) {
+        p0?.execSQL(SQL_DELETE_ENTRIES)
+        onCreate(p0)
+    }
+
+    fun addNote(value : ContentValues): Boolean{
+        val db = writableDatabase
+        val result = db.insert(TABLE_NOTE,"",value)
+        db.close()
+        return result.toInt() != -1
+    }
+
+    fun getAllNote():MutableList<Note>{
+        val notes = mutableListOf<Note>()
+        val db = readableDatabase
+        val cursor = db.query(TABLE_NOTE,null, null, null, null, null,null,null)
+        if(cursor.moveToFirst()){
+            do{
+                notes.add(Note(cursor))
+            }while (cursor.moveToNext())
+        }
+        cursor.close()
+        db.close()
+        return notes
+    }
+
+    fun updateNote(value: ContentValues, id: Int): Boolean{
```

Comment:
```suggestion
    fun updateNote(id: Int, value: ContentValues): Boolean{
```
Để như này thì sẽ dễ hiểu hơn: Update note ở id nào, dữ liệu là gì